README.md

```diff
@@ -10,7 +10,7 @@ To use this provider, you will need the following:
 
 * [Terraform] 0.13+
 * A Hydra instance running [commit
-`6e53767`](https://github.com/NixOS/hydra/commit/6e537671dfa21f89041cbe16f0b461fe44327038)
```

Comment:
Should we say *at least* (old commit) but *recommended* (new commit)? this commit we're replacing is truly the first commit we support, but after that are good bug fixes that may not be totally critical. I dunno, maybe it is a bit silly to even suggest they use a revision it is buggy with, but I don't want them seeing they're on 6e5... and not on 67b... and decide they can't use it.

Reply:
Added a `NOTE`, PTAL?